src/secp256k1.c

```diff
@@ -247,8 +247,15 @@ static int secp256k1_pubkey_load(const secp256k1_context* ctx, secp256k1_ge* ge,
     } else {
```

Comment:
I think we should simply get rid of this branch entirely, see https://github.com/bitcoin-core/secp256k1/issues/1352. I can work on a PR next week.

Reply:
See #1480

Reply:
Yes thanks! I considered doing this but didn't want to make this PR dependent on a separate issue.

Reply:
Got rid of this by rebasing.